int32 instance_id
---
bool result
